--- c-results/raw/cvode/serial/cvRoberts_block_klu.stdout
+++ rust-results/raw/cvode/serial/cvRoberts_block_klu.stdout
@@ -13,10 +13,10 @@
 group 0: At t = 4.0000e+06      y =  5.167122e-04    2.067905e-09    9.994833e-01
-group 0: At t = 4.0000e+07      y =  5.201745e-05    2.080808e-10    9.999480e-01
-group 0: At t = 4.0000e+08      y =  5.210816e-06    2.084338e-11    9.999948e-01
-group 0: At t = 4.0000e+09      y =  5.188603e-07    2.075442e-12    9.999995e-01
-group 0: At t = 4.0000e+10      y =  5.387257e-08    2.154903e-13    9.999999e-01
+group 0: At t = 4.0000e+07      y =  5.201742e-05    2.080807e-10    9.999480e-01
+group 0: At t = 4.0000e+08      y =  5.210867e-06    2.084358e-11    9.999948e-01
+group 0: At t = 4.0000e+09      y =  5.181280e-07    2.072513e-12    9.999995e-01
+group 0: At t = 4.0000e+10      y =  4.249183e-08    1.699673e-13    1.000000e+00
 
 Final Statistics:
-nst = 543    nfe = 788    nsetups = 115    nje = 12
-nni = 785    nnf = 4      netf = 30        ncfn = 0      nge = 0
+nst = 561    nfe = 812    nsetups = 113    nje = 11
+nni = 809    nnf = 3      netf = 30        ncfn = 0      nge = 0
 
