--- c-results/raw/idas/serial/idasSlCrank_FSA_dns.stdout
+++ rust-results/raw/idas/serial/idasSlCrank_FSA_dns.stdout
@@ -7,11 +7,11 @@
 
-Number of steps                    = 233
-Number of residual evaluations     = 1180
-Number of Jacobian evaluations     = 46
-Number of nonlinear iterations     = 720
+Number of steps                    = 231
+Number of residual evaluations     = 1131
+Number of Jacobian evaluations     = 42
+Number of nonlinear iterations     = 711
 Number of error test failures      = 0
-Number of nonlinear conv. failures = 26
-Number of step solver failures     = 1
+Number of nonlinear conv. failures = 24
+Number of step solver failures     = 0
 --------------------------------------------
-  G =       3.3366155611545363
+  G =       3.3366157997761721
 --------------------------------------------
@@ -27,3 +27,3 @@
 ---------------BACKWARD------------------
-   dG/dp:    3.3345e-01  -3.6375e-01
+   dG/dp:    3.3344e-01  -3.6375e-01
 -----------------------------------------
