--- c-results/raw/arkode/C_serial/ark_analytic_lsrk.stdout
+++ rust-results/raw/arkode/C_serial/ark_analytic_lsrk.stdout
@@ -21,7 +21,7 @@
 Final Statistics:
-Current time                  = 10.0040469322476
-Steps                         = 1120
-Step attempts                 = 1124
+Current time                  = 10.0047469441809
+Steps                         = 1121
+Step attempts                 = 1125
 Stability limited steps       = 232
-Accuracy limited steps        = 1124
+Accuracy limited steps        = 1125
 Error test fails              = 4
@@ -31,4 +31,4 @@
 Last step size                = 0.01791
-Current step size             = 0.0370361796227445
-RHS fn evals                  = 141974
+Current step size             = 0.0375123165381938
+RHS fn evals                  = 142049
 Number of dom_eig updates     = 1
@@ -39,2 +39,2 @@
 
-ACCURACY at the final time   = 1.52101e-13
+ACCURACY at the final time   = 1.09246e-13
