--- c-results/raw/arkode/C_serial/ark_analytic_lsrk_domeigest__arkid.dom_eig_est_init_preprocess_iters_1_sundomeigestimator.max_iters_1.stdout
+++ rust-results/raw/arkode/C_serial/ark_analytic_lsrk_domeigest__arkid.dom_eig_est_init_preprocess_iters_1_sundomeigestimator.max_iters_1.stdout
@@ -25,8 +25,8 @@
 Final Statistics:
-Current time                  = 10.0214604735918
-Steps                         = 1221
-Step attempts                 = 1284
-Stability limited steps       = 42
-Accuracy limited steps        = 1284
-Error test fails              = 63
+Current time                  = 10.0082585185898
+Steps                         = 1234
+Step attempts                 = 1286
+Stability limited steps       = 49
+Accuracy limited steps        = 1286
+Error test fails              = 52
 NLS step fails                = 0
@@ -34,13 +34,13 @@
 Initial step size             = 1.93010111094261e-10
-Last step size                = 0.0232720725963147
-Current step size             = 0.030027828635957
-RHS fn evals                  = 132483
-Number of dom_eig updates     = 86
-Number of fe calls for DEE    = 154
-Number of iterations for DEE  = 96
+Last step size                = 0.0248992496771087
+Current step size             = 0.0252928329774622
+RHS fn evals                  = 132338
+Number of dom_eig updates     = 78
+Number of fe calls for DEE    = 136
+Number of iterations for DEE  = 88
 Max. num. of stages used      = 200
 Max. num. of stages allowed   = 200
-Max. spectral radius          = 1010099.31616951
+Max. spectral radius          = 1010099.89814833
 Min. spectral radius          = 1009899
 
-ACCURACY at the final time   = 3.45945e-13
+ACCURACY at the final time   = 8.63754e-13
