--- c-results/raw/arkode/C_serial/ark_analytic_lsrk_domeigest.stdout
+++ rust-results/raw/arkode/C_serial/ark_analytic_lsrk_domeigest.stdout
@@ -25,8 +25,8 @@
 Final Statistics:
-Current time                  = 10.0177477358601
-Steps                         = 1221
-Step attempts                 = 1284
-Stability limited steps       = 47
-Accuracy limited steps        = 1284
-Error test fails              = 63
+Current time                  = 10.0149870240255
+Steps                         = 1234
+Step attempts                 = 1285
+Stability limited steps       = 44
+Accuracy limited steps        = 1285
+Error test fails              = 51
 NLS step fails                = 0
@@ -34,13 +34,13 @@
 Initial step size             = 1.93010111094261e-10
-Last step size                = 0.0232720722665437
-Current step size             = 0.0277471696137175
-RHS fn evals                  = 132465
-Number of dom_eig updates     = 86
-Number of fe calls for DEE    = 298
-Number of iterations for DEE  = 182
+Last step size                = 0.0253981963665305
+Current step size             = 0.0261648951666516
+RHS fn evals                  = 132185
+Number of dom_eig updates     = 77
+Number of fe calls for DEE    = 258
+Number of iterations for DEE  = 164
 Max. num. of stages used      = 200
 Max. num. of stages allowed   = 200
-Max. spectral radius          = 1010099.33048286
+Max. spectral radius          = 1010100.64516181
 Min. spectral radius          = 1009899.00000001
 
-ACCURACY at the final time   = 5.80647e-13
+ACCURACY at the final time   = 7.84706e-13
